jenkins/automation/commit-tag.groovy

```diff
@@ -18,7 +18,7 @@ pipeline {
             steps {
                 script { build_stage = env.STAGE_NAME }             
                 script {
-                    checkout([$class: 'GitSCM', branches: [[name: 'main']], doGenerateSubmoduleConfigurations: false, extensions: [], submoduleCfg: [], userRemoteConfigs: [[credentialsId: 'cortx-admin-github', url: 'https://github.com/Seagate/cortx-re']]])                
```

Comment:
Please make `CORTX_RE_REPO` and `CORTX_RE_BRANCH` as input parameters so that we can test this job with any of the custom repo details.

Reply:
@shailesh-vaidya, Done